docs/release/trg-8/trg-8-07.md

```diff
@@ -0,0 +1,122 @@
+
+### Security updates
+
+To enable Dependabot for security updates, you can leverage GitHub's Security tab. Go to the "Security" tab in your repository and follow the prompts to enable automated security updates.
+More information:  
+<https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates>
+
+### Version updates
+
+To enable Dependabot for version updates, create a dependabot.yml file in .github directory the root of your repository. In order to reduce number of generated bump Pull Requests, recommendation is to change default interval to i.e. weekly, as well as limit open PRs. See provided example below.
+
+### Example
+
+This configuration checks for Maven, GitHub Action and Docker updates on a weekly basis and creates pull requests for up to 5 updates at a time.
+
+:::caution
+
+Be careful, Dependabot PR merge can lead to out of date DEPENDENCIES file.
+
+Make sure DEPENDENCIES file is updated by DASH tool.
+
+:::
+
+```yaml
+version: 2
+updates:
+    # Maintain dependencies for Maven
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
+    # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
+    # Maintain dependencies for Docker
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
+
+ # Maintain dependencies for npm
+
+version: 2
+
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+version: 2
+updates:
+  # maintain dependencies for GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monday"
+    open-pull-requests-limit: 5
+    labels:
+      - "dependencies"
+      - "github_actions"
+
+  # maintain dependencies for Gradle
+  - package-ecosystem: "gradle" # checks build.gradle(.kts) and settings.gradle(.kts)
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    labels:
+      - "dependencies"
+      - "java"
+```
+:::tip
+
+You can change **interval** and **open-pull-requests-limit** based on your needs
+
+:::
+
+More information:
+<https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates>
+<https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file>
+
+:::info
+
+**Importance of Implemented Tests:**
+
+Ensure that your project has comprehensive test coverage. Automated tests are crucial for quickly validating that updates do not introduce regressions or break existing functionality.
+
+:::

```

Comment:
Hi @klaudiaZF 

I noticed that the document for Dependabot you shared bears a resamblance to one I created earlier [link](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/588) with some confusing modifications which Im not sure I agree with. Coincidentally there is aalready a [pull request](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/659) in the review state to move it from Draft to Active state, there is also a suggestion and it might be appropriate to place it in Security section.

I believe in maintaining transparency and fairness in our collaborative efforts hence propose to remove it from this PR and once review is complete and agreement in place I will make it happen to have it in security section.

Reply:
Hi @tomaszbarwicki 

I will leave Dependabot topic and wait for you to let us know if it's ready, is it ok for you?

Reply:
@tomaszbarwicki 
I propose adding Dependabot to the Security TRGs, as it primarily addresses security concerns. This consolidation would centralize our security-related guidance, making it more accessible and easier to manage. The existing Dependabot TRG is well-crafted and could be seamlessly integrated.